docs/fleet/provision.md

```diff
@@ -189,21 +187,19 @@ If you did not include a `viam-server` configuration file on your machine, or no
 By default, the hotspot network is named `viam-setup-HOSTNAME`, where `HOSTNAME` is replaced with the hostname of your machine.
 The WiFi password for this network is `viamsetup` by default.
 
-You can customize these values in the `agent_config` configuration object in your `viam-server` configuration file.
+You can customize these values in the `agent` configuration object in your `viam-server` configuration file.
 For example, to set the hotspot password to `acme123`, you can use the following configuration:
 
 ```json {class="line-numbers linkable-line-numbers"}
 ...
-"agent_config": {
-    "subsystems": {
-      "agent-provisioning": {
+"agent": {
+    "agent-provisioning": {
+      ...
+       "attributes": {
+        "hotspot_password": "acme123"
         ...
-        "attributes": {
-          "hotspot_password": "acme123"
-          ...
-        }
-      }
-    }
+       }
+   }
 }
 ```
 
```

Comment:
There might be some additional fragment-specific changes that could be part of this PR / ticket. I'd clarify with James that:
- I think the alert below, lines 245-249 is no longer true after this change (i.e. users can indeed configure agent settings using a fragment.
- Similarly, the alert at the end of the intro section to https://docs.viam.com/fleet/configure-a-fleet/ is also no longer true: the agent is now configurable via fragments.

I wonder if there is still a limitation whereby some Agent setup steps must still be manually configured (like some Agent config might already need to be in place before using a [provisioning conf file](https://docs.viam.com/fleet/provision/#use-a-provisioning-configuration-file) to add a fragment config of the rest of agent config settings, maybe?). Not sure, James would know!